src/PresentationalComponents/Battery/battery.js

```diff
@@ -6,30 +6,89 @@ import classNames from 'classnames';
 import './battery.scss';
 
 /**
- * This is a dumb component that only recieves properties from a smart component.
- * Dumb components are usually functions and not classes.
- *
- * @param props the props given by the smart component.
+ * This is the battery component that generates a 'battery'
+ * which corresponds to a level 1-4
+ * 1 - low, green - best case scenario
+ * 2 - medium, yellow
+ * 3 - high, orange
+ * 4 - critical, red - worst case scenario
+ * Also accepts a label which can be made invisible
  */
 
-const Battery = ({severity, label, labelHidden, className, ...props}) => {
-    let severityClass = classNames(
+const Battery = ({ severity, label, labelHidden, className, ...props }) => {
+
+    let batteryWrapperClasses = classNames(
+        className,
+        'ins-battery'
+    );
+
+    let batteryClasses = classNames(
         { [`ins-battery-${severity}`]: severity !== undefined }
     );
-    if(!labelHidden) {
+
+    let ariaLabels = {};
+    if (labelHidden) {
+        ariaLabels = { ['aria-label']: severity + ' ' + label };
+    }
+
+    function generateBattery (severity, batteryClasses, ariaLabels) {
+        return (
+            <i className= { batteryClasses } {...ariaLabels}>
+                <svg
+                    version="1.1"
+                    id="battery_svg"
+                    x="0px" y="0px"
+                    viewBox="0 0 448 512"
+                    style={{ enableBackground: 'new 0 0 448 512' }}
+                    shapeRendering= 'geometricpresision'>
+                    { batteryLevels(severity) }
+                </svg>
+            </i>
+        );
+    }
+
+    function batteryLevels (severity) {
+        switch (severity) {
+            case 'critical':
+            case 4:
+                return <path d="M16,212h416c8.8,0,16-7.2,16-16v-40c0-8.8-7.2-16-16-16H16c-8.8,0-16,7.2-16,16v40C0,204.8,7.2,212,16,212z M16,372h416
```

Comment:
Can you break these into their own components? Something like `CriticalBateryIcon`, `HighBateryIcon`, `WarnBateryIcon` and `InfoBateryIcon`.

This way we can do changes to just one icon and won't be bothered about the others.